src/formats/xhtml.js

```diff
@@ -720,6 +727,19 @@
 					tagName = '!comment';
 				}
 
+				if (nodeType === 1) {
+					// skip empty nlf elements (new lines automatically
+					// added after block level elements like quotes)
+					if (is(node, '.sceditor-nlf')) {
+						if (!firstChild || (!IE_BR_FIX &&
+							node.childNodes.length === 1 &&
+							/br/i.test(firstChild.nodeName))) {
+							// Mark as empty,it will be removed by the next code
+							empty = true;
+						}
```

Comment:
Thinking about it, if the `sceditor-nlf` node isn't empty then it should probably remove the class. So something like:

```js
else {
	node.classList.remove('sceditor-nlf');

	if (!node.className) {
		removeAttr(node, 'class');
	}
}
```